lib/shelley/src/Cardano/Wallet/Shelley/Network/Node.hs

```diff
@@ -374,6 +376,10 @@ withNodeNetworkLayerBase tr net np conn versionData tol action = do
             fetchRewardAccounts tr queryRewardQ
         , timeInterpreter =
             _timeInterpreter (contramap MsgInterpreterLog tr) interpreterVar
+        , eraHistory = do
+            i <- atomically (readTMVar interpreterVar)
+            let W.StartTime t0 = getGenesisBlockDate
```

Comment:
We _could_ replace our own `StartTime` with `Cardano.Slotting.Time.SystemStart` from `cardano-base`. We are getting our `SlotNo` from there.